api/src/main/java/org/openmrs/module/attachments/AttachmentsContext.java

```diff
@@ -189,6 +189,17 @@ public Encounter getAttachmentEncounter(Patient patient, Visit visit, Provider p
 		return encounter;
 	}
 	
+	/*
+	 * return String array with global property values
```

Comment:
It would be nice if this was a full Javadoc commend.

```suggestion
	 * @return An array of comma-separated values for the named global property
```